### ISSUE TITLE
Creating instances with EasyRandom Library fails

### ISSUE THREAD

**Issue by arnzel** [immutables/immutables#1152]
Hi for some tests i like to create instances of Classes annotated with "@Immutable" and
"@Modifiable" with random property values. for this purpose i tried the following library
https://github.com/j-easy/easy-random in version 4.1.0

I create the instances like this

`
MyModel myTestModel = EASY_RANDOM.nextObject(MyModel.class);
`

**Comment by elucash** [comment]
I expect this is because of easyrandom library using reflection to instantiate an object and fill fields, bypassing internal tracking mechanisms. Modifiable objects are expected to be checked for correctness/invariants, so when setting required attributes using setters, the internal bits are being cleared so it knows that all were set. To workaround this you might try to either mark all attributes as `@Nullable` (can use any annotation with `Nullable` simple name) or try to switch validation to none `@Style(validationMethod = NONE)` (this from my memory, exact att can be that or similar).

**Comment by arnzel** [comment]
Hmm i dont want to deactivate the validation in general. Is there no other way ? Or any other approach to archieve the goal i described ?

**Comment by elucash** [comment]
can the library use `setXyz` methods and not fields?, that would immediately solve the problem

**Comment by arnzel** [comment]
Hmm i dont know if the library can. But it would be nice if i could turn off validation for classes annotated with @Immutable or @Modifiable  in some way programmatically.

**Comment by elucash** [comment]
Basically, while you're already using quite a hackish reflection (via easy-random lib) to set random values, you can write static utility using reflection to set `initBits` field to `0` (after creating with the randomization factory), signaling that "all is ok". It will not cover all the machinery (`lazyBits` for `@Lazy` attributes and some other exotic stuff), but I guess it can help with up and running.

(very large objects with more than 64 fields will also have `initBits1`, `initBits2` etc, however, chances are you will never have such objects)

**Comment by arnzel** [comment]
Yeah it might be a little bit hackish but we have some models with over 50 properties(directly or with embeded models) so i think other people maybe also have interest in this kind of feature :) 

I will try next time when i have this problem your solution with setting initBits(1/2) to 0. Thanks for taking your time to answer this question @elucash

**Comment by arnzel** [comment]
I tried setting initBits to 0 and it works well
